JS/Board.js

```diff
@@ -6,9 +6,36 @@ class Board {
     this.completed = false
   }
 
+
+  /////this needs refactoring
```

Comment:
File an issue for something that needs refactoring rather than leaving it as a comment in the codebase -- if you want your partner to be aware of the fact that "maybe this is still a little sloppy", just put that  in the description of your PR or leave a line comment here like I'm doing with this review.